spec/ivapi/error_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+describe Ivapi::Error do
+  describe '.from_response' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.